src/assets/locales/el/error404.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line
+export default {
+    error_404: "Σφάλμα 404",
+    page_not_found: "Η σελίδα δεν βρέθηκε",
+    go_to_homepage: "ΠΗΓΑΙΝΕ ΣΤΗΝ ΑΡΧΙΚΗ ΣΕΛΙΔΑ",
```

Comment:
```suggestion
    go_to_homepage: "Πήγαινε στην αρχική σελίδα",
```

Reply:
Thanks for this suggestion Nikolas, I have committed the changes.